robot/2023/src/main/java/frc/robot/Path.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot;
+
+public class Path extends Robot
+{
+    //these may need to be floats depending on the encoder, I'll just wait and see though
+    double target;
+    double wait;
+    Robot bot;
+    public Path(double target, double wait, Robot bot)
+    {
+        this.target = target;
+        this.wait = wait;
+        this.bot = bot;
+    }
+    public void tick() {
+        bot.leftEncoder.getDistance();
+        bot.rightEncoder.getDistance();
+        // set something
+    }
+    //Since we're only going forward and backwards, we can just use the left encoder to get distance.
+    //If we implement turns, we'll have to use both
+    public boolean isDone() {
+        if(bot.leftEncoder.getDistance() >= 0.1 && bot.leftEncoder.getDistance() <= 0.1)
```

Comment:
Should also have a check for wait>0.  So something like:

boolean ret = false;
if(bot.leftEncoder.get...) { ret = true }
if(ret && wait > 0) { return false; }
return ret;

Reply:
Resolved